packages/dnb-eufemia/src/extensions/forms/Value/PostalCodeAndCity/PostalCodeAndCity.tsx

```diff
@@ -40,5 +33,17 @@ function PostalCodeAndCity(props: Props) {
   return <StringValue {...stringValueProps} />
 }
 
+function usePostalCodeAndCityValue(props: Props) {
+  let value = props.value ?? undefined
+
+  const valueProps = useValueProps(props)
+
+  if (valueProps.value) {
+    value = valueProps.value
+  }
+
+  return value
+}
```

Comment:
```suggestion
function usePostalCodeAndCityValue(props: Props) {
  return useValueProps(props).value ?? props.value
}
```

This should be equivalent with less code.